### PR TITLE
Remove unused import

### PR DESCRIPTION
**Pull request by njr0**
Remove unused `use threads` in code in Sec 49

### CODE REVIEW

File: src/exercises/day-3/safe-ffi-wrapper.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Rust has great support for calling functions through a _foreign function
 interface_ (FFI). We will use this to build a safe wrapper for the `libc`
-functions you would use from C to read the filenames of a directory.
+functions you would use from C to read the names of files in a directory.
```
Comment (mgeisler) [inline src/exercises/day-3/safe-ffi-wrapper.md:5]: Please take out this change. I think it has already been merged by now in another PR :smile:

**Review by mgeisler — CHANGES_REQUESTED** [review]

File: src/exercises/day-3/safe-ffi-wrapper.md (hunk 1 above)
Comment (njr0) [reply src/exercises/day-3/safe-ffi-wrapper.md:5]: I don't know how to do that.

Comment (mgeisler) [reply src/exercises/day-3/safe-ffi-wrapper.md:5]: You probably just need to merge `main` into your branch.

In the future, you should start new branches from the latest version on `main`. Lots of it is automatic if you use the little pencil icon in the top-right of any page to edit the page:

image

**Comment by njr0** [comment]
I took the branch from main. I'm sure it's all because I did the first pull request from main (I'd just directly edited there).

Even if I sync, and even though you seem to have merged the changes, it says my main is 4 commits ahead of the google one and I can't seem to fix that.

I'm tempted to delete my clone of the main repo, refork it and start from there. But I don't know whether (a) that can cause any actual problems or (b) whether it will at least break links if people try to see where things came from.

I feel there should be a way of saying "make mine like the upstream one", but I can't figure out how to do it.

Like I say, I'm not really used to pull requests...

**Comment by njr0** [comment]
Oh, and I don't have a little pencil in Emacs...

**Comment by njr0** [comment]
I'll just blow it away and start again...what could go wrong?

**Comment by mgeisler** [comment]
> Oh, and I don't have a little pencil in Emacs...

Right, me neither — if you use Emacs, I suggest using Magit and then you should quickly see how the history is shaped. You can then rebase the part of your branch you need for a pull request and quickly push it to GitHub.



> I'll just blow it away and start again...what could go wrong?

That's a fine approach for small changes like this: just make a new clone and do the changes there. Or do them in the GitHub editor like I suggest via the little pencil icon.

**Comment by mgeisler** [comment]
> Just blew it away and resubmitted two clean pull requests. Sorry, I'll get there...I've been using git forever, but on the shared repos I work on everyone just has commit access so the whole "pull request" workflow is relatively new to me.

I know that feeling! I've helped several people get used to the GitHub dance over the last few months :slightly_smiling_face: 
 
> I try to avoid doing things on the web whenever possible...(I know that might be anathema with Google!) The web's a scary place, you know!

Hehe, you would be surprised... we have a lot of die-hard terminal users here. We also have a lot of web stuff, but I'm old enough to remember a world before everything got crammed into the browser and I often miss those days where you could use fast local applications to work on your data. I'm super glad that I can still use Emacs for all my work — LSP and `rust-analyzer` has really be fundamental in enabling this since all editors can get good completion now.

**Pull request closed without merging**